src/community/jms-cluster/jms-geoserver/src/main/java/org/geoserver/cluster/impl/rest/ClusterResource.java

```diff
@@ -123,7 +123,7 @@ public JMSConfigHTMLFormat(Request request, Response response,
 
 		@Override
 		protected Configuration createConfiguration(Object data, Class clazz) {
-			Configuration cfg = new Configuration();
+			Configuration cfg = new Configuration(Configuration.VERSION_2_3_25);
```

Comment:
I am checking the Freemarker documentation, setting this flag seems to enable a lot of non backwards compatible behavioral changes... this needs to be carefully evaluated and documented for people upgrading GeoServer, or better, if possible, not done at all unless strictly necessary.
Is it possible to get the html security improvements without throwing in also all these changes?